### ISSUE TITLE
Create a slim image without osx/windows toolchains

### ISSUE THREAD

**Issue by moul** [multiarch/crossbuild#13]

**Comment by kohanyirobert** [comment]
While I was fiddling with #7 it came to me, that the right would be creating a specific box for different targets. No package clashes that way. Would mean more maintenance. Just throwing in the idea.

**Comment by moul** [comment]
Yes, this is clearly something to think!

When I created this image, I was building my binaries using different tools with different APIs, results, outputs, and I needed to maintain my build tools and monitor new breaking releases for each methods.
So, I decided to build a unique tool that can handle everything with the same API; I'm quite satisfied today about the way we can use this image with only one environment variable to change.

This image definitely needs to stay as a single big image; but I think a smaller image with only the Linux triples (about 60-70% smaller than the current image) may be useful.

For now I will close this issue and wait for someone to "wake up" this thread if the size become a real problem, so we can keep the repository simple and concentrate on real problems :)
